packages/web-components/src/components/ic-radio-group/test/basic/ic-radio-group.e2e.ts

```diff
@@ -306,8 +306,8 @@ describe("ic-radio-group component", () => {
     await page.waitForChanges();
 
     const value = await (
-      await page.find('input[name="test"]')
-    ).getProperty("value");
+      await page.findAll('input[name="test"]')
```

Comment:
I think, after removing the renderHiddenInput, you no longer need to use findAll & can leave these lines unaltered

Reply:
I believe it still needs findAll due to the fact that there are multiple ic-radio-options that its got to filter through, since each of the inputs would now be exposed